package-examples/ghost/configmap-mariadb-ini.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: mariadb
   namespace: example
 data:
-  my.ini: |-
+  my.cnf: |-
```

Comment:
I remember some discussion about keepting the extension to `ini`. Is it intended change ?

Reply:
yeah, I added the reason in the PR description. The `my.cnf` is [MariaDB default config file name](https://mariadb.com/kb/en/configuring-mariadb-with-option-files/#default-option-file-locations-on-linux-unix-mac)